src/pet.js

```diff
@@ -0,0 +1,78 @@
+minAge = 0;
+minHunger = 0;
+maxFitness = 10;
+
+function Pet(name) {
+    this.name = name;
+    this.age = minAge;
+    this.fitness = maxFitness;
+    this.hunger = minHunger;
+};
+
+Pet.prototype = {
+    get isAlive() {
+        return this.age < 30 && this.hunger < 10 && this.fitness > 0;
+    }
+};
+
+Pet.prototype.growUp = function() {
+        ageIncrements = 1;
+        fitnessIncrements = 3;
+        hungerIncrements = 5;
+
+        this.age += ageIncrements;
+        this.fitness -= fitnessIncrements;
+        this.hunger += hungerIncrements;
+        
+        if(!this.isAlive) {
+        throw new Error('Your pet is no longer alive')
+    }
+    };
```

Comment:
The indentation is off here. The opening bracket should be at the same level as the closing bracket, and each time you go into another layer inside it should only be indented by one layer, so here:
```
Pet.prototype.growUp = function() {
    ageIncrements = 1;
    fitnessIncrements = 3;
    hungerIncrements = 5;

    this.age += ageIncrements;
    this.fitness -= fitnessIncrements;
    this.hunger += hungerIncrements;
        
    if (!this.isAlive) {
        throw new Error('Your pet is no longer alive')
    }
};
```
Also, if you wanted to improve the maintainability of the work here and in the test file you could move the variables you create here to be with the rest of the constants in their own file, which you could then import into the tests to reference there too